app/controllers/AppealsController.scala

```diff
@@ -160,7 +160,13 @@ class AppealsController @Inject()(appConfig: AppConfig,
                     }
                     if (!seqOfNotifications.isEmpty) {
                       logger.debug(s"[AppealsController][submitAppeal] Posting SDESNotifications: $seqOfNotifications to Orchestrator")
-                       fileNotificationOrchestratorConnector.postFileNotifications(seqOfNotifications)
+                      fileNotificationOrchestratorConnector.postFileNotifications(seqOfNotifications).map {
+                        response =>
+                          response.status match {
+                            case INTERNAL_SERVER_ERROR =>
+                              logger.info(s"[AppealsController][submitAppeal] - INTERNAL_SERVER_ERROR from postFileNotifications with body: ${response.body}")
+                          }
```

Comment:
This will cause a MatchError if the response is anything except ISE (500). Also when an error has occurred, you want to log at the ERROR level so that it can be filtered if needs be. Be careful with the content, if I didn't know what postFileNotifications did then I would have to look at the code, if we can point to where these issues are coming from it will help any investigations

You may want to do something like:
```suggestion
                          response.status match {
                            case OK =>
                              logger.debug(s"[AppealsController][submitAppeal] - Received OK from file notification orchestrator")
                            case status =>
                              logger.error(s"[AppealsController][submitAppeal] - Received unknown response (${status}) from file notification orchestrator. Response body: ${response.body}")
                          }
```